### PR TITLE
Bulk operations on events

### PR DESCRIPTION
**Pull request by vied12**
- Allows to select one or many events
- Bulk planning creation
- Bulk event spiking

### CODE REVIEW

File: client/actions/agenda.js [hunk 1]
```diff
@@ -134,14 +134,19 @@ const fetchAgendaById = (_id) => (
  * @param {object} agenda - The agenda to add the planning to
  * @return Promise
  */
-const _addPlanningToAgenda = ({ planning, agenda }) => (
+const _addPlanningsToAgenda = ({ plannings, planning, agenda }) => (
```
Comment (ioanpocol) [inline client/actions/agenda.js]: Can you keep only plannings and apply the same pattern as on line 168?

File: client/actions/tests/agenda_test.js [hunk 2]
```diff
@@ -349,10 +349,10 @@ describe('agenda', () => {
                     notify,
                     $timeout,
                 })
-                .then((planning) => {
-                    expect(planning).toEqual(plannings[0])
-                    // Cannot check dispatch(addPlanningToAgenda()) using a spy on dispatch
-                    // As addPlanningToAgenda is a thunk function
+                .then((plannings) => {
+                    expect(plannings).toEqual([plannings[0]])
```
Comment (ioanpocol) [inline client/actions/tests/agenda_test.js]: Is this correct?

Comment (vied12) [inline client/actions/tests/agenda_test.js]: I think so. Nothing has changed on the planning. Only the agenda has been updated.

Comment (ioanpocol) [inline client/actions/tests/agenda_test.js]: Before it was planning and plannings and now it's plannings on both sides.

**Pull request merged**